code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -2,12 +2,12 @@
 /// HYPOSPRAY
 ////////////////////////////////////////////////////////////////////////////////
 
-/obj/item/chems/hypospray //obsolete, use hypospray/vial for the actual hypospray item
+/obj/item/chems/hypospray
 	name = "hypospray"
 	desc = "A sterile, air-needle autoinjector for rapid administration of drugs to patients."
-	icon = 'icons/obj/syringe.dmi'
-	item_state = "hypo"
-	icon_state = "hypo"
+	icon = 'icons/obj/hypospray.dmi'
+	icon_state = ICON_STATE_WORLD
+	abstract_type = /obj/item/chems/hypospray
```

Comment:
Isn't it convention for `abstract_type` to come first, i.e. before `name =`? That makes it clearer that it's abstract.